aclocal.m4

```diff
@@ -561,9 +548,13 @@ AC_DEFUN([OCAML_QUOTED_STRING_ID], [
 ])
 
 AC_DEFUN([OCAML_CC_SUPPORTS_ATOMIC], [
-  AC_MSG_CHECKING([whether the C compiler supports _Atomic types])
-  saved_LIBS="$LIBS"
-  LIBS="$LIBS $1"
+  AC_MSG_CHECKING([for $CC options to support _Atomic types])
```

Comment:
Sorry to bike-shed this slightly further, but I think it's slightly more autoconf-y to include the option being tried here, and have the result still be yes/no (see similar checking for pthreads in build-aux/ax_pthread.m4); so (with this line moved below the computation of `$opts`, obviously):
```suggestion
  AC_MSG_CHECKING([if $CC support _Atomic types with ${opts:-no additional options}
```

Reply:
Ok! I tried to make mine look like `checking for CC options to support C11` and related, but I agree that printing options is nicer.